ui/src/reportAnIssue.ts

```diff
@@ -15,6 +15,8 @@ export function reportAnIssue(actionId: string, parsedError: IParsedError): void
     const { extensionName, extensionVersion } = getPackageInfo();
 
     const body: string = `
+<Please be sure to remove any private information before submitting.>
```

Comment:
`<` shows up in the 'Write' tab of GitHub comments, but doesn't show up in the 'Preview' tab or after publishing. Maybe change it to `&lt;`? Can you fix the ones below, too?

![screen shot 2018-08-29 at 2 05 19 pm](https://user-images.githubusercontent.com/11282622/44815519-9948e180-ab94-11e8-9b73-508934ba82ac.png)

![screen shot 2018-08-29 at 2 05 22 pm](https://user-images.githubusercontent.com/11282622/44815522-9b12a500-ab94-11e8-86fd-56362b0266f9.png)

Reply:
I had wondered, but assumed the current ones were working and thus correct.  Will do.